src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/404.cshtml.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Smart.FA.Catalog.Web.Pages;
+
+public class NotFoundPageModel : PageModel
+{
+    public const string ErrorTitleKey = nameof(ErrorTitle);
+
+    public const string ErrorMessageKey = nameof(ErrorMessageKey);
+
+    public string ErrorTitle { get; set; } = null!;
+
+    public string ErrorMessage { get; set; } = null!;
+
+    public PageResult OnGet()
+    {
+        SetTitle();
+        SetMessage();
+
+        return Page();
+    }
+
+    private void SetTitle()
+    {
+        if (TempData.ContainsKey(ErrorTitleKey))
+        {
+            ErrorTitle = TempData[ErrorTitle]!.ToString()!;
+        }
```

Comment:
```suggestion
        if (TempData.TryGetValue(ErrorTitleKey, out var errorTitleKey))
        {
            ErrorMessage = errorTitleKey!.ToString()!;
        }
```
(same for the other if/else statements)

Reply:
As stated i call wont be doing this.